File: app/src/main/java/com/example/test_loggin/Menu.java
package com.example.test_loggin;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.test_loggin.data.Usuario;

public class Menu extends AppCompatActivity {
    int id=0;
    Usuario u;
    daoUsuarios dao;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_menu);
        Bundle b=getIntent().getExtras();
        id=b.getInt("id");
        dao= new daoUsuarios(this);
        u=dao.getUsuarioById(id);
        
    }
}

File: app/src/main/java/com/example/test_loggin/Menu.kt
package com.example.test_loggin

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.test_loggin.data.Usuario

class Menu : AppCompatActivity() {
    var id = 0
    var u: Usuario? = null
    var dao: daoUsuarios? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_menu)
        val b = intent.extras
        id = b!!.getInt("id")
        dao = daoUsuarios(this)
        u = dao!!.getUsuarioById(id)
    }
}